Kitodo/src/main/resources/messages/messages_en.properties

```diff
@@ -172,8 +172,8 @@ choose=Choose
 chooseFile=Choose file
 citation.accessTimestamp=Accessed\:
 citation.containedIn=In\:
-citation.quotes.close=\u201D
-citation.quotes.open=\u201C
+citation.quotes.close=”
+citation.quotes.open=“
```

Comment:
```suggestion
citation.quotes.close="
citation.quotes.open="
```

Reply:
There are still two different quotation marks for German (and this would also be true for French), so only English now uses the same character for both.